avm/Cargo.toml

```diff
@@ -5,7 +5,7 @@ license = "AGPL-3.0"
 homepage = "https://alan-lang.org"
 documentation = "https://docs.alan-lang.org"
 repository = "https://github.com/alantech/alan"
-version = "0.1.34"
+version = "0.1.34-5"
```

Comment:
Can this be reverted?

Reply:
It can, but should it? That is actual prerelease that can be downloaded and it makes sense to know what prerelease main is on particularly for Daemon testing?

Reply:
Otherwise we actually have to delete tags and preleases and push those deletions to `main` which is somewhat weird to me

Reply:
Well, I'm not sure if the `.version.sh` script will barf on this when we try to bump the version next. Can you confirm it won't?

I personally would prefer that the versions are just not hand edited ever to prevent potential mistakes; this still has to be tagged appropriately for it to work right, for instance, so maybe there should be a mechanism to support pre-release versioning there as an explicit thing (that bypasses the `npm publish`, amongst other things).

Reply:
> Otherwise we actually have to delete tags and preleases and push those deletions to `main` which is somewhat weird to me

Those tags will already break the moment you merge the PR since the exact commit its tagging disappears?

Reply:
> > Otherwise we actually have to delete tags and preleases and push those deletions to `main` which is somewhat weird to me
> 
> Those tags will already break the moment you merge the PR since the exact commit its tagging disappears?

Ah that is true!

Reply:
Will put it back